docs/codeexamples/writing_edsl.cc

```diff
@@ -0,0 +1,524 @@
+
```

Comment:
Please move this file to `plaidml/edsl/edsl_docs.cc`.

This file also doesn't compile (as shown by our buildkite build logs)